onfido/onfido.py

```diff
@@ -18,3 +19,4 @@ def __init__(self, api_key, version='v2', requestor=None):
         self.Checks = Checks(self.requestor)
```

Comment:
I agree. However these make up the public API of the package and so it is outside the scope of this PR to change the global style.

Perhaps I will tidy this up at the weekend.